### DISCUSSION TITLE
mmpose with other mmdetection

### DISCUSSION THREAD

**Discussion by ghost** [open-mmlab/mmpose#2256] [Q&A]
Hi, there. I wanna use mmpose conbine mmdetection.
Conditions are video of mmpose 2D Hand Keypoints and mmdetection.
I make new python fileBut  an error occurred. Could you please tell me some advice??
![Screenshot from 2023-04-19 16-47-34](https://user-images.githubusercontent.com/126233717/233006051-643a138e-b982-4b38-af33-ef48be5308e8.png)

**Comment by Ben-Louis** [comment]
I think you could refer to https://github.com/open-mmlab/mmpose/blob/0.x/demo/docs/2d_hand_demo.md and try our demo script to perform hand keypoint detection using MMPose and MMDetction
